apps/web/src/components/HoverCard/HoverCardUsernameInner.tsx

```diff
@@ -126,13 +126,17 @@ export function HoverCardUsernameInner({ preloadedQuery }: Props) {
         )}
       </StyledCardHeaderContainer>
       {isLoggedIn && !isOwnProfile && <UserSharedInfo userRef={user} />}
-    </VStack>
+    </Section>
   );
 }
 
+const Section = styled(VStack)`
+  width: 100%;
+`;
+
 const StyledCardHeader = styled(HStack)`
   display: flex;
-
+  gap: 2px;
```

Comment:
nit: the component that this extends, `HStack`, takes `gap` as a property so you can pass this directly into `StyledCardHeader`!

```
<StyledCardHeader gap={2} ... />
```